metadata-ingestion/src/datahub/utilities/mapping.py

```diff
@@ -102,12 +104,21 @@ def process(self, raw_props: Dict[str, Any]) -> Dict[str, Any]:
                     operation = self.get_operation_value(
                         operation_key, operation_type, operation_config, maybe_match
                     )
+                    if operation_type == Constants.ADD_TERMS_OPERATION:
+                        # add_terms operation is a special case where the operation value is a list of terms.
+                        # We want to aggregate these values with the add_term operation.
+                        operation_type = Constants.ADD_TERM_OPERATION
+
                     if operation:
-                        if isinstance(operation, str):
+                        if isinstance(operation, str) or isinstance(operation, list):
                             operations_value_set = operations_map.get(
                                 operation_type, set()
                             )
-                            operations_value_set.add(operation)  # type: ignore
+                            if isinstance(operation, list):
+                                breakpoint()
+                                operations_value_set.update(operation)  # type: ignore
+                            else:
```

Comment:
check or assert that it is a list for future proofing?

Reply:
the if statements above should check this